DICUI/Tasks.cs

```diff
@@ -49,15 +49,16 @@ public class DumpEnvironment
         public string OutputFilename;
 
         // UI information
-        public char DriveLetter;
+        public Drive Drive;
         public KnownSystem? System;
         public MediaType? Type;
-        public bool IsFloppy;
         public string DICParameters;
 
         // External process information
         public Process dicProcess;
 
+        public bool IsFloppy { get => Drive.IsFloppy; }
```

Comment:
you can keep this in the same location where `public bool IsFloppy;` was, it looks cleaner that way